src/client/features/view/cy/createToolTip.js

```diff
@@ -0,0 +1,183 @@
+const h = require('hyperscript');
+const classNames = require('classnames');
+const tippy = require('tippy.js');
+
+
+//Manage the creation and display of tooltips
+//Requires a valid name, cytoscape element, and parsedMetadata array
+class metadataTip {
```

Comment:
classes should start with capitals e.g MetadataTip

Reply:
Renamed the class and also made the file name slightly more clearer.